image-layout.md

```diff
@@ -0,0 +1,47 @@
+## Open Container Initiative Image Layout Specification
+
+The OCI Image Layout is a slash separated layout of OCI content-addressable objects and human significant references (refs) 
+This layout MAY be used in a variety of different transport mechanisms from archive formats (e.g. tar, zip) or used in shared filesystem environments (e.g. nfs) or in networked file fetching systems (e.g. http, ftp, rsync).
+Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appopriate manifest from the manifest list, unpacking the filesystem serializations in the correct order, and then converting the image configuration into an OCI Runtime config.json.
+
+The image layout has two top level directories: 
+
+- "blobs" contains content-addressable blobs
```

Comment:
On Fri, May 27, 2016 at 12:28:05PM -0700, Stephen Day wrote:
> > +- "blobs" contains content-addressable blobs
>
> I'm not sure I have an opinion either way, I'll regret not bringing
> this up: we use the term "blobs" because these are free-form binary
> data. They are not "objects" because they have no required
> structure, other than that imposed by their access path (ie, through
> a descriptor).

I'm not sure what your comment is in response to, but I'm ok with
“objects” → “blobs” throughout this PR.